dcos-log/journal/reader/config.go

```diff
@@ -52,6 +52,23 @@ func OptionMatch(m []JournalEntryMatch) Option {
 	}
 }
 
+// OptionMatchOR is a functional option that filters entries and applies logical OR to user
+// arguments []JournalEntryMatch.
+func OptionMatchOR(m []JournalEntryMatch) Option {
+	return func(r *Reader) error {
+		if r.Journal == nil {
+			return ErrUninitializedReader
+		}
+
+		for _, match := range m {
+			r.Journal.AddMatch(match.String())
+			r.Journal.AddDisjunction()
```

Comment:
Is this the kind of thing where `AddDisjunction` only needs to be applied to slice indices [0..(n-1)) ?  In other words, will this possibly leave a dangling OR at the end of the match?